packages/notifier/src/storesub.js

```diff
@@ -115,3 +194,31 @@ export const makeStoredPublisherKit = (storageNode, marshaller, childPath) => {
     subscriber,
   };
 };
+
+/**
+ * Like makePublishKit this make a `{ publisher, subscriber }` pair for doing efficient
+ * distributed pub/sub supporting both "each" and "latest" iteration
+ * of published values.
+ *
+ * What's different is this tees records, writing out to storageNode.
+ *
+ * @template [T=unknown]
+ * @param {ERef<StorageNode>} [storageNode]
+ * @param {ERef<Marshaller>} [marshaller]
+ * @param {string} [childPath]
+ * @returns {PublishKit<T>}
+ */
+export const makeStoredPublishKit = (storageNode, marshaller, childPath) => {
+  const { publisher, subscriber } = makePublishKit();
```

Comment:
On master, [`makePublishKit`](https://github.com/Agoric/agoric-sdk/blob/4cca744382fc465492153a5e9d4919edefe9c637/packages/notifier/src/publish-kit.js#L203) is differentiated from `makeEmptyPublishKit` (the former requiring an initial value, the latter not). Given the presence of such variation, I think this function is a little too specialized and it would probably be better to drop it in favor of callers handling publish kit creation and wiring into chain storage on their own, e.g.
```js
const { publisher, baseSubscriber } = makeEmptyPublishKit(); // or makePublishKit(initialValue)
const storedSubscriber = makeStoredSubscriber(baseSubscriber, storageNode, marshaller);
// Protect publisher, share storedSubscriber.
```

Reply:
https://github.com/Agoric/agoric-sdk/pull/5704/commits/060dbf257139a9c407e8e1538620cea7b82f96e8 removed `makeEmptyPublishKit` because all our publish kits are to start empty

This function is a convenience for what contracts have to do a lot of. If you request that it move out`@agoric/notifier` I'll move it to run-protocol and not export it.

Reply:
Yes, that may be a more reasonable starting point.